wurst/systems/units/hostiles/BabyAnimal.wurst

```diff
@@ -0,0 +1,25 @@
+package BabyAnimal
+import ID
+import BonyAnimal
+import Orders
+import GeometryUtils
+import ClosureTimers
+
+let SPAWN_WANDER_DISTANCE = 100.
+
+public abstract class BabyAnimal extends BonyAnimal
+    construct(unit whichUnit)
+        super(whichUnit)
+
+    construct(player whichPlayer, vec2 pos)
+        super(whichPlayer, pos)
+
+    construct(player whichPlayer, vec2 pos, angle facing)
+        super(whichPlayer, pos, facing)
+
+    override function postCreate()
+        super.postCreate()
+        getUnit().issuePointOrderById(Orders.move, getPos().polarOffset(randomAngle(), SPAWN_WANDER_DISTANCE))
+        getUnit().addAbility(GHOST_VIS_ID) //Add ghost (visible)
```

Comment:
I think comment is redundant. Where is the ID package coming from?

Reply:
Built-in to wurst. Comment removed

Reply:
So that isn't actually in the ID package, which is where I thought it was. Do you know what the ID package is for?

I would add the visible Ghost ability [here](https://github.com/wurstscript/WurstStdlib2/blob/200f171296d017d262aec8ad67b3962cdedd4e67/wurst/_wurst/assets/AbilityIds.wurst#L128).

Where you're getting it now seems to be [here](https://github.com/wurstscript/WurstStdlib2/blob/0aa2173acff1ba53fa56d70821c9e86fdcfb0548/wurst/_wurst/Basics.wurst#L21). But the `Basics` package isn't imported so I'm not sure how you're using it.